File: Data Structures & Algorithms/majority-element/submission-0.java
class Solution {
    public int majorityElement(int[] nums) {
        int counter = 0;
        int val = 0;
        for(int i=0; i<nums.length; i++){
            val = nums[i];
            for (int j=0; j<nums.length; j++){
                if(val == nums[j]){
                    counter++;
                    if(counter>nums.length/2){
                        break;
                    }
                }
            }
        }
        return val;
    }
}

File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        res = set()
        for num in nums:
            count = sum(1 for i in nums if i == num)
            if count > len(nums) // 3:
                res.add(num)
        return list(res)